elastic/network/cnp-egress-elastic-tcp-protocol.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: cilium.io/v2
+kind: CiliumNetworkPolicy
+metadata:
+   name: cnp-egress-elastic-tcp-protocol
+  namespace: default  #change with your namespace
+spec:
+  description: "Allow tcp egress connections via port 443"
+  endpointSelector:
+    matchLabels:
+      app: elastic-operator   #change with your app
+  egress:
+  - toPorts:
+    - ports:
+      - port: "443"
+        protocol: TCP
+    toCIDR:
+    - 35.202.179.126/32   #change with your CIDR 
```

Comment:
Remove IP from the Policy @AbhinavCSY
cc: @raviknox

Reply:
Committed the change @salman-accuknox